packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -322,6 +351,21 @@
   </GridPopover>
 {/if}
 
+{#if relationshipFields && !focused}
+  <GridPopover {open} {anchor}>
+    <div class="relationship-fields">
+      {#each Object.entries(relationshipFields) as [fieldName, fieldValue]}
+        <div class="relationship-field-name">
+          {fieldName}
+        </div>
+        <div class="relationship-field-value">
+          {fieldValue}
```

Comment:
NAB: I think we should add a min width here to ensure we can at least render a few words for the value before line wrapping, then also use a line clamp of 3 lines to prevent values getting too long. You'll also need a few new styles to ensure long values get properly truncated with ellipses properly.